modules/openapi/tests/src/ModelWrapper.scala

```diff
@@ -0,0 +1,155 @@
+/* Copyright 2022 Disney Streaming
+ *
+ * Licensed under the Tomorrow Open Source Technology License, Version 1.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    https://disneystreaming.github.io/TOST-1.0.txt
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package smithytranslate.compiler.openapi
+
+import software.amazon.smithy.model._
+import software.amazon.smithy.model.node._
+import scala.jdk.CollectionConverters._
+import software.amazon.smithy.build.transforms.FilterSuppressions
+import software.amazon.smithy.build.TransformContext
+import software.amazon.smithy.model.shapes.SmithyIdlModelSerializer
+import software.amazon.smithy.model.transform.ModelTransformer
+import software.amazon.smithy.model.traits.Trait
+import software.amazon.smithy.model.shapes.Shape
+import software.amazon.smithy.model.traits.BoxTrait
+import software.amazon.smithy.diff.ModelDiff
+import java.util.stream.Collectors
+
+// In order to have nice comparisons from test reports.
+class ModelWrapper(val model: Model) {
```

Comment:
Bringing this over from smithy4s dynamic tests since the one we were using previously was prone to false-positive reports of differences since it relied on the smithy `Model#equals` method

Reply:
good idea